app/models/service_template.rb

```diff
@@ -234,12 +234,20 @@ def set_ownership(service, user)
     service.save
   end
 
-  def self.default_provisioning_entry_point
-    '/Service/Provisioning/StateMachines/ServiceProvision_Template/default'
+  def self.default_provisioning_entry_point(service_type = 'atomic', _prov_type = nil)
+    if service_type == 'atomic'
+      '/Service/Provisioning/StateMachines/ServiceProvision_Template/CatalogItemInitialization'
+    elsif service_type == 'composite'
+      '/Service/Provisioning/StateMachines/ServiceProvision_Template/CatalogBundleInitialization'
+    end
+  end
+
+  def self.default_retirement_entry_point(_service_type = 'atomic', _prov_type = nil)
+    '/Service/Retirement/StateMachines/ServiceRetirement/CatalogItemInitialization'
```

Comment:
@gmcculloug @bzwei and I discussed service entry points a few weeks ago. 
There are three service provisioning instances besides the CatalogBundleInitialization:
default, clone_to_service, and catalogiteminitization. They all differ only in the "pre" provision states.
Default - no pre states
clone_to_service - sequencer only pre state
CatalogItemInitialization - sequencer, dialog parser, and catalogiteminitialization

Since the Default and clone_to_service instances don't handle dialog parsing,   
I think it makes sense to alway execute the sequencer, parser and initialization in the CatalogItemInitialization instance.